src/styles.ts

```diff
@@ -10,7 +10,9 @@ export default createGlobalStyle`
 
   body {
     background: ${
-      window.location.pathname === '/cadastro' ? 'var(--purpleBackground)' : ''
+      window.location.pathname === '/cadastro' || '/editarUsuario'
```

Comment:
No caso esse operador desse jeito faz com que se o `window.location.pathname === '/cadastro'` for falsy ele retorne o '/editarUsuario' aí vocês fazem o ternário e sempre vai cair na primeira condição do ternário, pq se for falsy em cima ele já retorna uma string não vazia.

Acho que o que vcs queriam era isso:

```ts
window.location.pathname === '/cadastro' || window.location.pathname === '/editarUsuario'
```

Reply:
Juro que tinhamos alterado isso